style_editor.html

```diff
@@ -2390,10 +2396,10 @@
   extraButtons(arg) {
     if (arg == 1) return "";
     var id = this.get_id();
-    var ret = "<button onclick='DuplicateLayer("+id+","+arg+",event)'>+</button>";
-    ret += "<button onclick='RemoveLayer("+id+","+arg+",event)'>&#10799;</button>";
-    if (arg > 2) ret += "<button onclick='UpLayer("+id+","+arg+",event)'>&#5169;</button>";
-    if (arg <= this.LAYERS.length) ret += "<button onclick='DownLayer("+id+","+arg+",event)'>&#5167;</button>";
+    var ret = "<button id='extra_buttons' class='extra-buttons' title='Duplicate Layer' onclick='DuplicateLayer("+id+","+arg+",event)'>+</button>";
```

Comment:
What's the point of giving the button an ID if it's the same as a bunch of other buttons?

Reply:
Leftover from initial edit where there was styling via IDs.  Removed.